src/preview.ts

```diff
@@ -23,6 +23,7 @@ function updateDimensions(iframe: HTMLIFrameElement) {
     'style',
     `${iframeStyle}; width: ${scrollWidth}px; height: ${scrollHeight}px;`
   );
+  iframe.setAttribute('id', 'chromatic-e2e-playwright-inner-iframe');
```

Comment:
We should drop the `playwright` from here because the iframe is not framework-specific.